### PR TITLE
Show checkboxes for task list items.

### PR DESCRIPTION
**Pull request by dcampbell24**
This file was used for testing: https://github.com/dcampbell24/mdBook/blob/djot-support/test_djot_book/src/individual/task.dj

### CODE REVIEW

**Comment by dcampbell24** [comment]
Why do you git ignore the file that is failing? Should I just force add it?

**Comment by hellux** [comment]
> Why do you git ignore the file that is failing? Should I just force add it?

The *.rs test files are automatically generated by their corresponding *.test files. The djot_js_task_lists ones that are currently failing are from the [reference implementation repository](https://github.com/jgm/djot.js) so they cannot be changed here directly. Preferably, the reference implementation should be updated first and then the submodule can point to a newer revision.

**Comment by dcampbell24** [comment]
Okay, thanks for the information.

File: src/html.rs [hunk 1]
```diff
@@ -574,6 +573,15 @@ impl<'s, 'f> Writer<'s, 'f> {
                     Container::Math { display } => {
                         out.write_str(if *display { r#">\["# } else { r#">\("# })?;
                     }
+                    Container::TaskListItem { checked } => {
+                        out.write_str(">\n")?;
+                        self.indent(&mut out)?;
```
Comment (hellux) [inline src/html.rs:578]: Need to also handle when using minified, there should be no whitespace in that case. Should be able to use `self.block(&mut out, 0)` to conditionally write a newline. It also accounts for the indentation.

**Comment by hellux** [comment]
Mind squashing and rebasing on top of master before I merge?

**Comment by hellux** [comment]
Looks good, thanks!

**Pull request merged**